dev-lang/luau/luau-0.645.ebuild

```diff
@@ -22,17 +22,33 @@ fi
 
 LICENSE="MIT"
 SLOT="0"
-
+IUSE="test"
```

Comment:
not needed, since you did not add any test dependencies, USE=test is only added if we need additional test deps